src/TimeoutMigrationTool.RabbitMq.Tests/RabbitMqTimeoutCreatorTest.cs

```diff
@@ -0,0 +1,99 @@
+namespace TimeoutMigrationTool.RabbitMq.Tests
```

Comment:
Should the project be named `.IntegrationTests` see https://github.com/Particular/TimeoutMigrationTool#tests ?